Client/ClientRoom.hpp

```diff
@@ -53,6 +53,9 @@ namespace client_data
         /// @brief Init some entities only created by the client.
         void _initEntities();
 
+        /// @brief Check if the player still alive and update the gamestate if is dead
+        void _holdGameOver(void);
```

Comment:
No need to specify the parameter `void`

Reply:
I am not okay with that, i used the C convention

Reply:
But we're using cpp, you should remove it